### PR TITLE
Fix mathjax configuration

### PR DESCRIPTION
**Pull request by Discordius**

### CODE REVIEW

File: packages/lesswrong/server/editor/utils.ts [hunk 1]
```diff
@@ -16,6 +16,10 @@ Utils.trimEmptyLatexParagraphs = (dom) => {
           len--;
       }
   }
+  const [firstLatexElement] = dom.getElementsByClassName("mjx-chtml");
+  const styleNode = dom.createElement("style");
+  styleNode.innerHTML = css;
```
Comment (jimrandomh) [inline packages/lesswrong/server/editor/utils.ts:21]: Change this to `innerText`. I don't see a specific vulnerability here, but it wouldn't be super surprising if there were a LaTeX program that would control part of the stylesheet, which would be XSS.

Comment (Discordius) [inline packages/lesswrong/server/editor/utils.ts:21]:
```suggestion
  styleNode.innerText = css;
```

Comment (Discordius) [reply packages/lesswrong/server/editor/utils.ts]: Alas, when I replaced it with innerText it broke and stopped working. Not sure why.

Comment (Discordius) [reply packages/lesswrong/server/editor/utils.ts]: `textContent` worked though

**Pull request merged**